cmd/nerdctl/main.go

```diff
@@ -52,152 +48,124 @@ func xmain() error {
 		return logging.Main(os.Args[2])
 	}
 	// nerdctl CLI mode
-	return newApp().Run(os.Args)
+	return newApp().Execute()
+}
+
+func initializeConfig(cmd *cobra.Command) {
+	viper.SetEnvPrefix("CONTAINERD")
+	viper.AutomaticEnv()
```

Comment:
I'm not sure we want this.

In the previous revision, we only had three `CONTAINERD_*` env vars: `CONTAINERD_ADRESS`, `CONTAINERD_NAMESPACE`, and `CONTAINERD_SNAPSHOTTER`.
(And these var names were shown in --help)